File: backend-exam/src/main/java/com/backend_exam/backend_exam/dtos/QuestionDTO.java
package com.backend_exam.backend_exam.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class QuestionDTO {
    private String content;
    private String answerA;
    private String answerB;
    private String answerC;
    private String answerD;
    private String answerCorrect;
    private Long examId;
}

File: frontend/frontend-exam/src/app/dtos/question.dto.ts
import { ExamDto } from './exam.dto';
export interface QuestionDto {
  questionId: number;
  examId: number;
  content: string;
  answerA: string;
  answerB: string;
  answerC: string;
  answerD: string;
  answerCorrect: string;
  exam?: ExamDto;
}
